### PR TITLE
dist/build-aci.sh: build an aci

### PR DESCRIPTION
**Pull request by alban**
This patch depends on:
- actool patch-manifest:
  https://github.com/appc/spec/pull/258
- docker2aci on local files:
  https://github.com/appc/docker2aci/issues/32

### CODE REVIEW

**Comment by alban** [comment]
Update on the dependencies:
- actool patch-manifest ppc/spec#258 is merged :)
- docker2aci on local files appc/docker2aci#32 is waiting for review on https://github.com/appc/docker2aci/pull/37

**Comment by jonboulle** [comment]
@eyakubovich good for this to go?

**Comment by eyakubovich** [comment]
Looks good but can we add an option to either enable or disable this. The script is called `build-docker.sh` and if there're issues with aci conversion, I'd still like to be able to make releases without commenting out lines in the script.

**Comment by jonboulle** [comment]
Hmm, maybe this should be `dist/build-aci.sh` and just `source build-docker.sh` inside it?

**Comment by eyakubovich** [comment]
That would be fine as well.

**Comment by jonboulle** [comment]
@alban could you please take a look at this so we can close out coreos/rkt#389?

**Comment by alban** [comment]
Patch updated.

File: dist/build-aci.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+set -e
+
+if [ $# -ne 1 ]; then
+	echo "Usage: $0 tag" >/dev/stderr
+	exit 1
+fi
+
+tag=$1
+
+tgt=$(mktemp -d)
```
Comment (jonboulle) [inline dist/build-aci.sh]: unused?

Comment (alban) [inline dist/build-aci.sh]: You're right... I meant to save the .docker file in $tgt.
I'm fixing this.

**Comment by alban** [comment]
Branch updated.

**Comment by jonboulle** [comment]
LGTM, @eyakubovich can you ack/merge please?

**Comment by eyakubovich** [comment]
LGTM

**Pull request merged**